tasks/Receptionist.tex

```diff
@@ -30,8 +30,7 @@ \subsection*{Setup}
 	\begin{itemize}
 	\item \textbf{Host:} The host's name and favorite drink will be announced before the test. The host is already sitting in the living room.
 
-	\item \textbf{Guests:} Both guests have a name and favorite drink. An arriving guest will either step in front of the robot or ring the bell if the door is closed. Guests have to be guided to the living room to be introduced. Each of the guest will arrive separately. The sound of the bell will be provided to teams
-	during setup period before competition.
+	\item \textbf{Guests:} Both guests have a name and favorite drink. An arriving guest will either step in front of the robot or knock on the door if the door is closed. Guests have to be guided to the living room to be introduced. Each of the guest will arrive separately.
```

Comment:
For future reference. What is the reason to switch to knocking?

Reply:
We don't rely on the sound setup on site to work consistently for every test run